apps/comps/components/agent-profile/index.tsx

```diff
@@ -249,7 +250,7 @@ export default function AgentProfile({
                 Agent Rank
               </span>
               <span className="text-secondary-foreground mt-1 w-full text-left text-sm">
-                Not rated yet
+                {DISABLE_LEADERBOARD ? "TBA" : "Not rated yet"}
```

Comment:
```suggestion
                {DISABLE_LEADERBOARD ? "TBA" : agent.stats.rank}
```

ooh let's actually change the `"Not rated yet"` to the actual global rank